pkgs/top-level/python-packages.nix

```diff
@@ -29288,4 +29288,47 @@ in modules // {
       sha256 = "14kb9gxw39zhvrijhp066b4bm6bgv35iw56c394y4dyczpha0dij";
     };
   };
+
+  sqlalchemy_bb8 = buildPythonPackage rec {
```

Comment:
I wouldn't put `_bb8` in these. If they're only useful to BB8, then they might be better off in its nix file. If they're more broadly useful, then I'd take the suffix off.